src/rpc/blockchain.cpp

```diff
@@ -909,6 +924,24 @@ UniValue verifychain(const UniValue& params, bool fHelp)
     return CVerifyDB().VerifyDB(Params(), pcoinsTip, nCheckLevel, nCheckDepth);
 }
 
+UniValue getexperimentalfeatures(const UniValue& params, bool fHelp)
```

Comment:
These changes belong in `src/rpc/misc.cpp`, as they aren't associated with the block chain itself.

Reply:
Moved.